source/includes/_overview.md

```diff
@@ -2,18 +2,27 @@
 
 Welcome to the Hyperledger Iroha API docs!
 
-<iframe width="560" height="315" style="padding-left: 28px" src="https://www.youtube.com/embed/Hf_5j0b6TZM" frameborder="0" allowfullscreen></iframe>
+The documentation describes the endpoints for users to send transactions to peer network, containing one or many commands to perform allowed actions in the system, and also make queries to know the current state. Users are authenticated by their keypair and account id passed as the parameter with their transactions or queries.
```

Comment:
Looks like documentation describes a bit more than just endpoints how to send data to Iroha. Also I'm not sure note about account id should be here.
May be just move this paragraph after one about elevator pitch?

Reply:
The primary purpose of this website is to tell about endpoints and how to build the system. Later, we will migrate every information not relevant to endpoints to RTD. I will paraphrase and remove id in account id.